client/vscode/src/inlineCompletionActivation.ts

```diff
@@ -109,6 +111,14 @@ export class CodeWhispererInlineCompletionItemProvider implements InlineCompleti
         const firstCompletionDisplayLatency = Date.now() - requestStartTime
         // Add completion session tracking and attach onAcceptance command to each item to record used decision
         list.items.forEach((item: InlineCompletionItemWithReferences) => {
+            // POC-NEP: Set isInlineEdit flag if this is an edit suggestion
+            // This is the key to enabling the edit suggestion UI in VSCode
+            if (item.isInlineEdit) {
+                ;(item as any).isInlineEdit = true
```

Comment:
Isn't this redundant with line 116?  Or is this to force it to true from a truthy value?